targets/tools/js2c.py

```diff
@@ -77,10 +81,16 @@ def removeWhitespaces(code):
 OUT_PATH = './source/'
 SRC_PATH = './js/'
 
+parser = argparse.ArgumentParser(description="js2c")
+parser.add_argument('build_type', help='build type', default='release', nargs='?')
```

Comment:
We can restrict the choices to `release` and `debug` if we provide the `choices` list for the argument.